libs/shared/src/types/feature-flags/flags.types.ts

```diff
@@ -17,4 +17,16 @@ export type IFlagKey = `IS_${Uppercase<string>}_ENABLED`;
  */
 export function testFlagEnumValidity<TEnum extends IFlags, IFlags = Record<IFlagKey, IFlagKey>>(
   testEnum: TEnum & Record<Exclude<keyof TEnum, keyof IFlags>, ['Key must follow `IFlagKey` format']>
-) {}
+) {
```

Comment:
Option 2:
Explicit testing of match between the key and value for the enum, works for all enums provided, only shows failure when tests get run.